android/app/src/main/java/com/zingo/RPCModule.kt

```diff
@@ -18,7 +18,8 @@ class RPCModule internal constructor(private val reactContext: ReactApplicationC
         const val TAG = "RPCModule"
     }
 
-    private val LIGHTWALLETD_URL = "https://lwdv3.zecwallet.co";
```

Comment:
these ports are different likely, zecwallet.co uses 443 and zuul uses 9067 ... I don't think switching the hardcode for zuul is a great idea even though it has been running OK for a while - it's on sqlite! Not sure if lightwalletd.com and zecwallet.co are also onn sqlite and it doesnt matter ... but, I would hope those who go grants used postgres for the deployment ...

best thing would be to implement a switcher and default to the server with the tallest height ...

Reply:
It's something temporary and It's for coherence between the repo and the Apps for testing... nothing more. Of course this is not ideal....